### PR TITLE
make more pixel-perfect

### PR DESCRIPTION
**Pull request by hoppenichu**

### CODE REVIEW

File: CHTCollectionViewWaterfallLayout.m [hunk 1]
```diff
@@ -6,6 +6,7 @@
 //
 
 #import "CHTCollectionViewWaterfallLayout.h"
+#import "tgmath.h"
```
Comment (chiahsien) [inline CHTCollectionViewWaterfallLayout.m]: What is it for?

Comment (hoppenichu) [inline CHTCollectionViewWaterfallLayout.m]: For 64bit support.
http://stackoverflow.com/questions/5352457/cgfloat-based-math-functions/5352779#5352779

**Comment by chiahsien** [comment]
Thanks!

**Comment by hoppenichu** [comment]
Thank you for your review.

**Pull request merged**